src/components/CreditCardCVCField/CreditCardCVCField.tsx

```diff
@@ -28,7 +28,7 @@ const CreditCardCVCField: React.FC<Props> = ({ value, onChange, error, ...props
       value={value}
       onChange={formatCVC}
       pattern="\d*"
-      placeholder="cvc/cvv"
+      placeholder="123"
```

Comment:
What do you think about just using `###` as the placeholder?

Reply:
also, we can validate the `CVC` number:
e.g. only `Amex` card type can have 4 digits `CVC`
```
Payment.fns.validateCardCVC('123'); //=> true
Payment.fns.validateCardCVC('123', 'amex'); //=> true
Payment.fns.validateCardCVC('1234', 'amex'); //=> true
Payment.fns.validateCardCVC('12344'); //=> false
```

Reply:
We can do that.